sources/packages/web/src/services/http/dto/ScholasticStanding.dto.ts

```diff
@@ -24,10 +24,16 @@ export interface ScholasticStandingDataAPIInDTO {
   dateOfWithdrawal?: string;
 }
 
+/** Represents the scholastic standing submitted details. */
```

Comment:
Thanks for adding this comment. When using these type of comments try to do something like below, or use inline comments.
/**
 * Represents the scholastic standing submitted details.
 */